src/app/src/components/Timer.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { Box } from 'rebass';
+import { themeGet } from 'styled-system';
+
+import Text from './Text';
+import { saveSecondsToCompleteQuiz } from '../actions';
+
+const StyledTimer = styled(Box)`
+    position: absolute;
+    background-image: linear-gradient(
+        to right,
+        ${themeGet('colors.teals.0')} 30%,
+        ${themeGet('colors.teals.1')} 70%,
+        ${themeGet('colors.teals.2')} 100%
+    );
+    opacity: 11%;
+    right: 0px;
+    text-align: center;
+    width: 300px;
+    height: 50px;
+`;
+
+class Timer extends Component {
+    constructor() {
+        super();
+        this.state = {
+            elapsedSeconds: 0,
+        };
+    }
+
+    componentDidMount() {
+        this.interval = setInterval(
+            () =>
+                this.setState(state => ({
+                    elapsedSeconds: state.elapsedSeconds + 1,
+                })),
+            1000
+        );
+    }
+
+    componentWillUnmount() {
+        const { dispatch } = this.props;
+        const { elapsedSeconds } = this.state;
+
+        dispatch(saveSecondsToCompleteQuiz(elapsedSeconds));
+        clearInterval(this.interval);
+    }
+
+    render() {
+        const { elapsedSeconds } = this.state;
+        const minutes = Math.floor(elapsedSeconds / 60);
+        const seconds = elapsedSeconds % 60;
+
+        const formattedMins = ('0' + minutes).slice(-2);
+        const formattedSecs = ('0' + seconds).slice(-2);
+
+        return (
+            <StyledTimer>
+                <Text variant='large'>
+                    {formattedMins}:{formattedSecs} min
```

Comment:
Do we want to start the label as `secs` and then switch to `min` once the timer hits `1:00`? I was a little confused at first.

Reply:
That's a good question. I had been following the wires, but realize with your question that there's only examples of mm:ss off which I'd made this assumption. Calling in design! @alexelash  

<img width="274" alt="Screen Shot 2019-07-02 at 9 20 41 AM" src="https://user-images.githubusercontent.com/10568752/60515948-b06fc880-9caa-11e9-84e2-a97aa8dd2640.png">

Reply:
Alex is on vacation through RRP this week, so I'm going to merge this PR and add this comment to the Quiz style pass list.